tests/helpers/app-fixture.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
+
+var fs = require('fs-extra');
+var path = require('path');
+var symlinkOrCopySync = require('symlink-or-copy').sync;
+var merge = require('ember-cli-lodash-subset').merge;
+
+var fixturify = require('fixturify');
+var quickTemp = require('quick-temp');
+
+var originalWorkingDirectory = process.cwd();
+var root = path.resolve(__dirname, '..', '..');
+
+var PackageCache = require('../../tests/helpers/package-cache');
+var CommandGenerator = require('../../tests/helpers/command-generator');
+
+var ember = new CommandGenerator(path.join(root, 'bin', 'ember'));
+
+function AppFixture(name) {
+  this.type = 'app';
+  this.command = 'new';
+  this.name = name;
+  this._installedAddons = [];
+
+  this._init();
+}
+
+AppFixture.prototype = {
+
+  _init: function() {
+    process.chdir(root);
```

Comment:
this seems pretty odd, I'm guessing its due to quickTemp API?  I'd prefer to simply avoid that and generate our own tmp dirs...